csrc/scheduler/matmul.cpp

```diff
@@ -704,16 +692,6 @@ void scheduleFusionInputsForEpilogue(
 void scheduleMatmul(Fusion* fusion, const MatmulParams& params) {
   FusionGuard fg(fusion);
 
-  // Make sure we don't have global memory set on intermediate tensors from
-  // fusion segmentation
-  scheduler_utils::clearMemorySpace(fusion);
-
-  // Cache inputs
-  scheduler_utils::cacheInputs(fusion, true);
-
-  // Cache and fork outputs
-  auto cached_outputs = scheduler_utils::cacheAndForkOutputs(fusion, true);
```

Comment:
Trust but verify: is it safe to move this after CombineMulSum?

Reply:
Right. They should be set before itself. Undid this change.